File: src/main/java/isop/api/ControllerExpression.java
package isop.api;

import isop.Build;

/**
 * Created by mathieu.
 */
public class ControllerExpression {
    private String controllerName;
    private Build build;
    public ControllerExpression(String controllerName, Build build){
        this.controllerName = controllerName;
        this.build = build;
    }
    public ActionControllerExpression action(String actionName)
    {
        return new ActionControllerExpression(controllerName, actionName, build);
    }
}

File: src/main/java/isop/api/ControllerExpression.kt
package isop.api

import isop.AppHostBuilder

/**
 * Created by mathieu.
 */
class ControllerExpression(private val controllerName: String, private val build: AppHostBuilder) {
    fun action(actionName: String): ActionControllerExpression {
        return ActionControllerExpression(controllerName, actionName, build)
    }
}
